### PR TITLE
Find or make a video for Advanced Command Line topics

### LINKED ISSUE #513: Find or make a video for Advanced Command Line topics
The [Advanced Command Line topic outline](https://github.com/Techtonica/curriculum/blob/master/command-line/command-line-advanced.md) is not quite finished.  Please help us complete:

- [x] Outline Section: Guided Practice
- [x] Outline Section: Challenge
- [x] Outline Section: Check for Understanding
- [x] Proofread topic outline
- [ ] Find or make video, add link to "materials" section in outline
- [x] Find or make slideshow about lesson, add link to "materials" section in outline

For FAQ on contributing, see: https://github.com/Techtonica/curriculum#volunteering

### PR DESCRIPTION
**Pull request by Soumya0803**
Added link to the video. 
Fixes #513

### CODE REVIEW

**Comment by manaswinidas** [comment]
@Soumya0803 Please mention the issue you are addressing or fixing.

**Comment by Soumya0803** [comment]
@manaswinidas I have updated it in the PR.

**Comment by manaswinidas** [comment]
Please do that in the description so that the issue is automatically referenced

**Comment by Soumya0803** [comment]
@manaswinidas Updated the issue number in the description.

**Comment by alodahl** [comment]
@Soumya0803 this link just takes me to a channel. Please provide a link to video that demonstrates the skills being taught in this topic’s outline and slideshow.

File: command-line/command-line-advanced.md [hunk 1]
```diff
@@ -41,6 +41,7 @@ Teach people some handier tools for advanced file searching and processing
 - [Filenames and Pathnames in Shell: How to do it Correctly](https://www.dwheeler.com/essays/filenames-in-shell.html)
 - [An Awk Primer/Awk Command-Line Examples](https://en.wikibooks.org/wiki/An_Awk_Primer/Awk_Command-Line_Examples)
 - [I/O Redirection](http://wiki.bash-hackers.org/syntax/redirection)
+- [Videos](https://www.youtube.com/playlist?list=PL6gx4Cwl9DGCkg2uj3PxUWhMDuTw3VKjM)
```
Comment (alodahl) [inline command-line/command-line-advanced.md]: @Soumya0803 this link just takes me to a channel. Please provide a link to 1 or 2 videos that demonstrate the specific skills being taught in this topic’s outline and slideshow.

**Comment by alodahl** [comment]
@soumya0803 are you still able to work on this issue?

File: command-line/command-line-advanced.md (hunk 1 above)
Comment (Soumya0803) [reply command-line/command-line-advanced.md]: @alodahl I found a playlist which consists of 14 videos on command line.
Link: https://www.youtube.com/watch?v=bE9DyH43C2I&list=PLVqGqrTs4ZWOhcApSWYIX_rnPMZDAClJa&index
Please review this link. 
I am also trying,if I could find 1 or 2 videos which covers the skills being taught.

Comment (alodahl) [reply command-line/command-line-advanced.md]: The materials section is required, and i can’t ask them to watch 1.5 hours of video. The other option is to do a screen recording and email it to techtonica@gmail.org, and send us the link in the comments here to review.

Comment (alodahl) [reply command-line/command-line-advanced.md]: You can put these videos under “supplemental materials”; we just can’t require 14 videos.

**Comment by alodahl** [comment]
@Soumya0803 if I don’t hear from you in a few days, I will give this issue to someone else.

**Comment by Soumya0803** [comment]
@alodahl Sorry I by mistake skipped your earlier comment.
I tried to search a lot but for the commands in the advanced command line section, I was not able to find a specific video. 
One video I found 
link:https://www.youtube.com/watch?v=NodKFuUocQY
But this does not cover all the commands.
Please, you may assign this to someone else.

**Comment by alodahl** [comment]
@Soumya0803 this is a good video!  Can you tell me which commands it doesn't cover?

**Comment by Soumya0803** [comment]
@alodahl Sure, I will add the link and the descrpion. The commands that I could find which are not covered:
- wc , sort, cut, xargs, apropos, awk, sed, wc, pushd, popd

**Comment by alodahl** [comment]
@Soumya0803 once you replace the video with your latest link(https://www.youtube.com/watch?v=NodKFuUocQY) , we can merge this.

**Comment by Soumya0803** [comment]
@alodahl I have updated the link and added the description.

**Pull request merged**